tests/test_populate_catalog.py

```diff
@@ -652,3 +655,26 @@ def should_add_link_to_root_catalog_in_series(
                 expected_root_catalog_links
                 == load(smart_open.open(root_url, mode="rb"))[STAC_LINKS_KEY]
             )
+
+
+@mark.infrastructure
+@patch("geostore.populate_catalog.task.read_file")
+def should_log_error_message_when_an_exception_is_raised_trying_to_update_catalog(
+    pystac_read_file_mock: MagicMock,
+) -> None:
+
+    error = Exception()
+    expected_message = f"{LOG_MESSAGE_LAMBDA_FAILURE}: Unable to populate catalog due to “{error}”"
+
+    pystac_read_file_mock.side_effect = error
+
+    with patch("geostore.populate_catalog.task.LOGGER.warning") as logger_mock:
+        with raises(Exception):
+            handle_message(any_s3_url())
+
+        logger_mock.assert_any_call(
+            expected_message,
+            extra={
+                GIT_COMMIT: get_param(ParameterName.GIT_COMMIT),
+            },
+        )
```

Comment:
FYI, Black will automatically split things into multiple lines if there's a trailing comma like on the two preceding lines. If you remove the trailing comma I believe all or most of this assertion should fit on a single line.